src/os/bluestore/BitAllocator.cc

```diff
@@ -1669,3 +1707,11 @@ void BitAllocator::free_blocks_dis(int64_t num_blocks, ExtentList *block_list)
   alloc_assert(get_used_blocks() >= 0);
   unlock();
 }
+
+void BitAllocator::dump()
+{
+  int count = 0;
+  serial_lock(); 
+  dump_int(count);
```

Comment:
Yeah, i didn't notice that in the previous iteration.
Would be good rename dump_int() something else, conflicts with dump_init() of formatter. 
Rest looks good.